src/components/UriViewer.js

```diff
@@ -176,7 +177,7 @@ const UriViewer = _.flow(
 
   const loadMetadata = async () => {
     try {
-      if (isGs(uri)) {
+      if (isGs(uri) && false) {
```

Comment:
The false seems suspicious, but given that this is draft perhaps you are just testing.

Reply:
yeah this was a fragment of this still being in draft -- will remove the tag once this is ready for review!